style.css

```diff
@@ -13,170 +13,479 @@
+}
 
-  .header {
-    min-height: 70px;
-    background: #fff;
-    width: 100%;
-  }
+.header {
+  min-height: 70px;
+  background: #fff;
+  width: 100%;
+}
 
-  .header-nav {
-    display: flex;
-    padding: 10px 20px;
-    align-items: center;
-    justify-content: space-between;
-  }
+.header-nav {
+  display: flex;
+  padding: 10px 20px;
+  align-items: center;
+  justify-content: space-between;
+}
 
-  .header-manu {
-    padding: 3px 5px;
-  }
+.header-manu {
+  padding: 3px 5px;
+}
 
-  .manu-icon {
-    width: 48px;
-    height: auto;
-  }
+.manu-icon {
+  width: 48px;
+  height: auto;
+}
 
-  .nav-items {
-    display: none;
-    list-style: none;
-    column-gap: 20px;
+.nav-items {
+  display: none;
+  list-style: none;
+  column-gap: 20px;
+}
+
+.nav-item-action {
+  font-size: 15px;
+  font-weight: 500;
+  color: #344563;
+  line-height: 20px;
+  padding: 10px 12px;
+  text-decoration: none;
+}
+
+.nav-item-action:hover {
+  color: #fff;
+  background-color: #344563 !important;
+}
+
+.mobile-nav-items {
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100vh;
+  display: none;
+}
+
+.article {
+  display: flex;
+  min-height: 100vh;
+  flex-direction: column;
+  background-color: #fff;
+  border-bottom-left-radius: 70px;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+  background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+}
+
+.article-item1 {
+  margin: 20% 12% 15%;
+}
+
+.headline-title {
+  color: #172b4d;
+  font-weight: 900;
+  font-size: 40px;
+  font-style: bolder;
+  line-height: 52px;
+  display: flex;
+  flex: none;
+  margin-bottom: 20px;
+  font-family: Poppins-bold, sans-serif;
+}
+
+.headline-content {
+  font-size: 16px;
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 30px;
+  flex: none;
+  margin-left: 0;
+  margin-right: 24px;
+}
+
+.headline-container {
+  display: grid;
+  row-gap: 25px;
+}
+
+.social-links {
+  display: flex;
+  list-style: none;
+  margin-top: 10px;
+  padding-left: 1%;
+  padding-right: 10px;
+}
+
+.social-icon {
+  padding: 0 10px;
+}
+
+.icon {
+  padding-left: 5px;
+}
+
+.social-title {
+  font-size: 16px;
+  font-weight: 600;
+  text-decoration: none;
+  color: #7f8cff;
+  line-height: 50px;
+  width: 120px;
+  height: 24px;
+}
+
+.work-section {
+  margin: 80px 0;
+  display: grid;
+  row-gap: 80px;
+  padding: 10px;
+  justify-items: center;
+}
+
+.work-card {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+}
+
+.work-preview {
+  display: flex;
+  align-self: center;
+  flex-direction: column;
+  justify-self: center;
+  overflow: hidden;
+}
+
+.work-preview img {
+  transition: transform 0.2s ease-in-out;
+  max-width: 100%;
+}
+
+.work-preview:hover .project-img {
+  transform: scale(1.4);
+}
+
+.work-detail {
+  align-self: center;
+  padding: 10px;
+}
```

Comment:
- [ ] To make your work details align just like your image from top to bottom, Kindly consider aligning your details `start` instead of `center` as that will make your design look like that of the `Figma` design and also help make your design look professional. 😊

Reply:
@Strangeal Thanks a lot for the Hints 👍. Changes have been made exact accordingly.